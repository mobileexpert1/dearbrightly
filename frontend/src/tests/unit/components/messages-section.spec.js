import React from 'react';
import { shallow } from 'enzyme';

import MessagesSection from '../../../sections/dashboard/pages/MessagesSection';

describe('MessagesSection', () => {
    it('should render correctly', () => {
        const component = shallow(<MessagesSection />);
        expect(component).toMatchSnapshot();
    });
});
