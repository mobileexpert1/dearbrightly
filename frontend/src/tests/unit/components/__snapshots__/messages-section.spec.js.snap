// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesSection should render correctly 1`] = `
<div
  className="messages-section-container"
>
  <div
    className="messages-section-main-content"
  >
    You have no messages at this time.
  </div>
</div>
`;
